### ISSUE TITLE
question

### ISSUE THREAD

**Issue by cavehuangly** [Mengqi-Lei/ConDSeg#12]
你好，为什么Kvasir-Sessile 的实验结果差距很大

**Comment by qyl-123** [comment]
我在ISIC2017复现结果大约差了4%

**Comment by slx1213** [comment]
想问一下.txt文件是如何生成的

**Comment by chendashuai-real** [comment]
> 你好，为什么Kvasir-Sessile 的实验结果差距很大

我也是，除了 SEG 这个数据集，其他几个差距有点大，不知道是不是划分不对

**Comment by Mengqi-Lei** [comment]
For the datasets Kvasir-SEG, GlaS, ISIC-2016, and ISIC-2017, we adopted the train/val/test division provided by the official of the original datasets. For Kvasir-Sessile, since the official did not set up the train/val division, we randomly divided it into train/val/test at a ratio of 156/20/20, which is a widely adopted division in other papers. Therefore, the possible reproduced results of Kvasir-Sessile may differ from those in our paper. However, for the other datasets with official divisions, as long as the experimental settings are correct, the performance of the paper should be relatively easily reproduced.

**Comment by wenchuanjiang** [comment]
Hello, may I ask how you handled the datasets that were split into three parts? Did you combine the validation set and the test set together as the final test set, or did you merge the validation set with the training set to form the final training set?

**Comment by mht-deep** [comment]
你好，可以给我Kvasir-Sessile数据集的train.txt,val.txt,test.txt,文件吗？谢谢
